corehq/apps/geospatial/tests/test_views.py

```diff
@@ -136,6 +136,7 @@ def test_config_update(self):
         self.assertEqual(config.user_location_property_name, 'some_other_name')
 
 
+@es_test(requires=[case_search_adapter], setup_class=True)
```

Comment:
Do we require case indexing for this test class? This class is only testing whether the view returns successfully. The tests for the paginated data itself is done in `TestGetPaginatedCasesOrUsers`.

Reply:
Apparently we do.
For one of the filters, we need the current case types in the domain, which calls `get_case_types_for_domain_es`.
Another way would be to mock the call and return an empty list.

Reply:
I would be in favour of simply mocking it out to remove the unnecessary overhead, however I don't have strong feelings about this and how it is implemented now is fine as well since it's required.

Reply:
Looking at the function es_test, it simply creates the index and then deletes it.
For this one, it will do it only once, for all tests in this class, since we have set it up on class.
I don't expect to put any different load than a mock of the call, so keeping the es decorator.